book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
```

Comment:
- [ ] a book has many rentals kindly set up the associations here so relationship between Person and Book using the intermediate class Rental
e.g:
```ruby
def add_rental(person, date)
   Rental.new(date, self, person)
end
```